src/iosrv/srv.c

```diff
@@ -841,11 +844,48 @@ dss_start_xs_id(int xs_id)
 static int
 dss_xstreams_init(void)
 {
+	char	*env;
 	int	rc;
 	int	i, xs_id;
 
 	D_ASSERT(dss_tgt_nr >= 1);
 
+	d_getenv_bool("SCHED_PRIO_DISABLED", &sched_prio_disabled);
```

Comment:
[Optional] Would be nicer to put some DAOS-ness into the env var names. E.g.,

    DAOS_SCHED_PRIO_DISABLED

or

    DSS_SCHED_PRIO_DISABLED

Reply:
Ok, I'll add DAOS prefix for all macros.